phpunit.xml.dist

```diff
@@ -20,6 +20,7 @@
     </coverage>
 
     <php>
+        <env name="COLUMNS" value="120"/>
```

Comment:
do we still need this?

Reply:
yes, we need this. otherwise we get line breaks in the middle of the sentence.